### PR TITLE
RM-7237 Do not add tooltip to validation error when DisableEditModeValidationMessages is set

### PR DESCRIPTION
**Pull request by a-ctor**
https://re-motion.atlassian.net/browse/RM-7237

### CODE REVIEW

File: Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditableRow.cs [hunk 1]
```diff
@@ -466,13 +466,15 @@ public IBusinessObjectBoundEditableWebControl[] GetEditControlsAsArray ()
           {
             ((HtmlControl)validationErrorMarker).Attributes["tabIndex"] = "0";
             ((HtmlControl)validationErrorMarker).Attributes["title"] = null;
-            ((HtmlControl)validationErrorMarker).Attributes[HtmlTextWriterAttribute2.AriaLabel] = toolTipBuilder.ToString();
+            if (!_editModeHost.DisableEditModeValidationMessages)
```
Comment (MichaelKetting) [inline Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditableRow.cs:469]:
```suggestion
            if (_editModeHost.DisableEditModeValidationMessages)
```

Please verify, I believe you got an negation error here.

File: Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditableRow.cs [hunk 2]
```diff
@@ -466,13 +466,15 @@ public IBusinessObjectBoundEditableWebControl[] GetEditControlsAsArray ()
...
+              ((HtmlControl)validationErrorMarker).Attributes[HtmlTextWriterAttribute2.AriaLabel] = toolTipBuilder.ToString();
           }
           else if (validationErrorMarker is WebControl)
           {
             ((WebControl)validationErrorMarker).TabIndex = 0;
             ((WebControl)validationErrorMarker).ToolTip = null;
-            ((WebControl)validationErrorMarker).Attributes[HtmlTextWriterAttribute2.AriaLabel] = toolTipBuilder.ToString();
+            if (!_editModeHost.DisableEditModeValidationMessages)
```
Comment (MichaelKetting) [inline Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditableRow.cs:476]:
```suggestion
            if (_editModeHost.DisableEditModeValidationMessages)
```

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

File: Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditableRow.cs (hunk 1 above)
Comment (a-ctor) [reply Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditableRow.cs:469]: Don't think so. We only want the tooltip if the validation messages have not been disabled -> `!DisableEditModeValidationMessages`

Comment (MichaelKetting) [reply Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditableRow.cs:469]: With the suggestion, there is no tooltip in 
image

With the original code, the tooltip is generated. Check also line ~506 where we render the validation message based on `DisableEditModeValidationMessages` being `false`. The updated code needs to compensate/use inverted semantics.

**Review by MichaelKetting — APPROVED** [review]

**Pull request merged**